src/Components/AdministratorPage/BureauExceptionList/BureauExceptionList.jsx

```diff
@@ -0,0 +1,70 @@
+import { useEffect } from 'react';
+import { withRouter } from 'react-router';
+import { useDispatch, useSelector } from 'react-redux';
+import Spinner from 'Components/Spinner';
+import ProfileSectionTitle from 'Components/ProfileSectionTitle';
+import Alert from 'Components/Alert';
+import { bureauExceptionBureauDataFetchData, bureauExceptionFetchData } from 'actions/bureauException';
+import BureauExceptionListCard from './BureauExceptionListCard';
+
+
+const BureauExceptionList = () => {
+  const dispatch = useDispatch();
+
+  const BureauExceptionDataLoading = useSelector(state => state.bureauExceptionsFetchDataLoading);
+  const BureauExceptionData = useSelector(state => state.bureauExceptions);
+  const BureauExceptionOptionsData = useSelector(state => state.bureauExceptionsOptions);
+  const BureauExceptionError = useSelector(state => state.bureauExceptionsFetchDataErrored);
+  const fetchAndSet = () => {
+    dispatch(bureauExceptionFetchData());
+    dispatch(bureauExceptionBureauDataFetchData());
+  };
+
+  useEffect(() => {
+    fetchAndSet(false);
```

Comment:
```suggestion
    fetchAndSet();
```

this function doesn't take any arguments ... you also prolly dont need to even abstract to its own function and can just fire the actions here